cardano-node/src/Cardano/Node/Configuration/POM.hs

```diff
@@ -242,6 +243,24 @@ instance FromJSON PartialNodeConfiguration where
              , npcShelleyGenesisFileHash
              }
 
+      parseAlonzoProtocol v = do
+        primary   <- v .:? "AlonzoGenesisFile"
+        secondary <- v .:? "GenesisFile"
```

Comment:
I made this mistake too. Apparently we don't need the `"GenesisFile"` field here as it was for backwards compatibility reasons.

Reply:
`GenesisFile` is removed.